services/py-api/tests/unit_tests/conftest.py

```diff
@@ -13,6 +13,8 @@
 from src.database.repository.teams_repository import TeamsRepository
 from src.database.transaction_manager import TransactionManager
 from src.server.schemas.request_schemas.schemas import ParticipantRequestBody
+from src.service.hackathon_service import HackathonService
+from src.service.participants_registration_service import ParticipantRegistrationService
 
 
```

Comment:
Somewhere here we can do a very brief explaination on how MagicMock(), AsyncMock() and just Mock() work, with links to the docs

Reply:
We can also say a word or 2 about conftest.py and its role in pytest

Reply:
https://github.com/AUBGTheHUB/monolith/blob/main/services/py-api/tests/unit_tests/README.md

Reply:
Updated the README in 2973c46fb277bc549a219ceb1d03bcceddbbb2f1